doc/introduction.rst

```diff
@@ -17,54 +17,114 @@
-* SVG friendly, to create and include vector titles and credits
-* Scrolling motion picture credits
-* Advanced Timeline (including Drag & drop, scrolling, panning, zooming, and snapping)
-* Frame accuracy (step through each frame of video)
-* Time-mapping and speed changes on clips (slow/fast, forward/backward, etc...)
-* Audio mixing and editing
-* Digital video effects, including brightness, gamma, hue, greyscale, chroma key (bluescreen / greenscreen) , and many more!
+OpenShot™ Video Editor has many abilities and features, including:
+
+- 2D animation support
+- 3D animated titles support
+- Advanced Timeline (including Drag & Drop, scrolling, zooming, and snapping)
+- Audio mixing and editing (multiple channel support)
+- Clip resizing, scaling, trimming, snapping, rotation, and cutting
+- Compositing, image overlays, watermarks
+- Cross-platform (Supports Linux, macOS, and Windows)
+- Desktop integration (Drag & Drop support)
+- Digital video effects (including brightness, gamma, hue, and gray-scale)
+- Frame accuracy (step through each frame of video)
+- Powerful curve-based Key frame animations
+- Scrolling motion picture credits
+- Support for many `video, audio, and image formats 
+  <https://en.wikipedia.org/wiki/FFmpeg#Supported_codecs_and_formats>`_
+- SVG friendly, to create and include vector titles and credits
+- Time-mapping and speed changes on clips (slow/fast, forward/backward, etc.)
+- Title templates, title creation, sub-titles
+- Unlimited tracks/layers for video editing
+- Video transitions with real-time previews
+- and many more!
 
 Screenshot
 ----------
-.. image:: images/ui-example.jpg
+
+.. image:: images/ui-example.png
 
 System Requirements
 -------------------
-Video editing benefits from large amounts of memory, modern CPUs, and fast disks. Basically,
-you want the best computer you can afford when editing video. Here are the minimum system
-requirements:
 
-* 64-bit Operating System (Linux, OS X, Windows 7/8/10)
-* Multi-core processor with 64-bit support
-* 4GB of RAM (16GB recommended)
-* 500 MB of hard-disk space for installation
-* Optional: Solid-state drive (SSD), if utilizing disk-caching (and an additional 10GB of
-  hard-disk space)
+Video editing uses sizable amounts of memory, modern multi-core CPUs, and 
+fast internal storage.  It is always best to have as much memory as possible.  
+You will also want a powerful processor.  You can manage using any processor 
+with a speed of 2.4 gigahertz (GHz) or higher.  Finally, you will want to make 
+sure you have plenty of storage available to hold the software and video 
+with which you will be working.  A Solid State Device (SSD) provides faster 
+disk access which helps when finalizing your project.  Most modern computers 
+should work well with OpenShot™.
+
+When editing video with OpenShot™, you want a computer system that meets these 
+**minimum system requirements**:
+
+- 64-bit Multi-core processor (2.4+ GHz)
+- 4 GB of Random-Access Memory (RAM)
```

Comment:
4 GB is still pretty low...

Maybe we should also have a recommended system requirements too:
16 GB RAM
64-bit Multi-Core processor (with 6 to 8 cores)
SSD with 500 GB
Modern NVidia card, with hardware accelerated video encoding/decoding
Monitor or laptop display capable of 1920x1080 (1080p) resolution